seatunnel-connectors-v2/connector-rocketmq/src/main/java/org/apache/seatunnel/connectors/seatunnel/rocketmq/source/RocketMqConsumerThread.java

```diff
@@ -59,6 +59,8 @@ public void run() {
             } catch (InterruptedException e) {
                 throw new RocketMqConnectorException(
                         RocketMqConnectorErrorCode.CONSUME_THREAD_RUN_ERROR, e);
+            } finally {
+                this.consumer.shutdown();
```

Comment:
public void run() {
        try {
            while (!Thread.currentThread().isInterrupted()) {
                try {
                    Consumer<DefaultLitePullConsumer> task = tasks.poll(1, TimeUnit.SECONDS);
                    if (task != null) {
                        task.accept(consumer);
                    }
                } catch (InterruptedException e) {
                    throw new RocketMqConnectorException(
                            RocketMqConnectorErrorCode.CONSUME_THREAD_RUN_ERROR, e);
                }
            }
        } finally {
            this.consumer.shutdown();
        }